ansible/README.md

```diff
@@ -0,0 +1,190 @@
+# Ansible playbook for Tendermint applications
+
+![Ansible plus Tendermint](img/a_plus_t.png)
+
+* [Prerequisites](#Prerequisites)
+* [Ansible setup](#Ansible setup)
+* [Running the playbook](#Running the playbook)
+
+The playbooks in this folder run [ansible](http://www.ansible.com/) roles which:
+
+* install and configure basecoin or ethermint
+* start/stop basecoin or ethermint and reset their configuration
+
+## Prerequisites
+
+* Ansible 2.0 or higher
+* SSH key to the servers
+
+Optional for DigitalOcean droplets:
+* DigitalOcean API Token
+* python dopy package
+
+Head over to the [Terraform folder](https://github.com/tendermint/tools/tree/master/terraform-digitalocean) for a description on how to get a DigitalOcean API Token.
+
+Optional for Amazon AWS instances:
+* Amazon AWS API access key ID and secret access key.
+
+The cloud inventory scripts come from the ansible team at their [GitHub](https://github.com/ansible/ansible) page. You can get the latest version from the contrib/inventory folder.
+
+## Ansible setup
+
+Ansible requires a "command machine" or "local machine" or "orchestrator machine" to run on. This can be your laptop or any machine that can run ansible. (It does not have to be part of the cloud network that hosts your servers.)
+
+Use the official [Ansible installation guide](http://docs.ansible.com/ansible/intro_installation.html) to install Ansible. Here are a few examples on basic installation commands:
+
+Ubuntu/Debian:
+```
+sudo apt-get install ansible
+```
+
+CentOS/RedHat:
+```
+sudo yum install epel-release
+sudo yum install ansible
+```
+
+Mac OSX:
+```
+sudo easy_install pip
+sudo pip install ansible
```

Comment:
Ok, according to blogs on the Internet ansible should be packaged with brew. (I don't have a Mac with brew lying around right now.) I'm going to assume that it's a recent version. I added it to the readme.